src/forms/renderers/OAuth.tsx

```diff
@@ -201,6 +278,17 @@ const OAuthproviderRenderer = ({
                     />
                 </Typography>
 
+                {isEdit ? (
+                    <Alert
+                        severity="info"
+                        sx={{
+                            maxWidth: '50%',
+                        }}
+                    >
+                        <FormattedMessage id="oauth.edit.message" />
+                    </Alert>
+                ) : null}
+
```

Comment:
Adding a new message to make it clear the user will need to re authenticate if they change the endpoint config.